packages/apps/human-app/frontend/src/api/fetcher.ts

```diff
@@ -153,32 +167,10 @@ export function createFetcher(defaultFetcherConfig?: {
       fetcherOptions.authenticated &&
       fetcherOptions.withAuthRetry
     ) {
-      let refetchAccessTokenSuccess: SignInSuccessResponse | undefined;
-      try {
-        refetchAccessTokenSuccess = await apiClient(
-          apiPaths.worker.obtainAccessToken.path,
-          {
-            successSchema: signInSuccessResponseSchema,
-            options: {
-              method: 'POST',
-              body: JSON.stringify({
-                // eslint-disable-next-line camelcase -- camel case defined by api
-                refresh_token: browserAuthProvider.getRefreshToken(),
-              }),
-            },
-          }
-        );
-        browserAuthProvider.signIn(
-          refetchAccessTokenSuccess,
-          browserAuthProvider.authType
-        );
-      } catch {
-        browserAuthProvider.signOut({ triggerSignOutSubscriptions: true });
-        return;
-      }
+      const refetchAccessTokenSuccess = await refreshToken();
 
       const newHeaders = appendHeader(fetcherOptionsWithDefaults, {
-        Authorization: `Bearer ${refetchAccessTokenSuccess.access_token}`,
+        Authorization: `Bearer ${refetchAccessTokenSuccess?.access_token}`,
```

Comment:
If success - it should be always defined and have `access_token`.
So there are two options (will depend on `refreshToken` implementation):
- if `refreshToken` throws in case of failed refresh - we should have `try..catch` and `return` in `catch` block (like it was)
- if `refreshToken` returns `null` in case of refresh not successful, we simply
```
const refreshTokenResult = await refreshToken();
if (!refreshTokenResult) {
  return;
}
```